wlauto/core/device.py

```diff
@@ -318,6 +318,10 @@ def capture_screen(self, filepath):
         """Captures the current device screen into the specified file in a PNG format."""
         raise NotImplementedError()
 
+    def capture_view_hierachy(self, filepath):
```

Comment:
this only makes sense for android devices, should not be in the device base class